packages/tools/u-boot/package.mk

```diff
@@ -6,7 +6,7 @@ PKG_NAME="u-boot"
 PKG_ARCH="arm aarch64"
 PKG_LICENSE="GPL"
 PKG_SITE="https://www.denx.de/wiki/U-Boot"
-PKG_DEPENDS_TARGET="toolchain swig:host"
+PKG_DEPENDS_TARGET="toolchain Python3:host swig:host"
```

Comment:
i think this is superseeded by `toolchain`.
because `Python3:host` is part of the `toolchain`. (`toolchain` > `toolchain:host` > `meson:host` > `Python3:host`)